runtime/doc/eval.txt

```diff
@@ -8248,6 +8251,11 @@ term_getaltscreen({buf})				*term_getaltscreen()*
 		{buf} is used as with |term_getsize()|.
 		{only available when compiled with the |+terminal| feature}
 
+term_getansicolors({buf})				*term_getansicolors()*
+		Get the ANSI color palette in use by terminal {buf}.
+		Returns a List of length 16 where each element is a String
+		representing a color in "#rrggbb" format.
```

Comment:
If {buf} is not given, it should default to the current buffer.  That needs to be noted here in the docs.

Reply:
Thanks, documented.  To be consistent with the other `term_` functions, `term_getansicolors('')` refers to the current buffer (`term_getansicolors('%')` also works).

Reply:
These functions (particularly `term_getansicolors()`) are currently returned as `[]`.  Is this the expected value (`[]`) at startup?

```bash
vim -g --clean
```
then
```vim
:verbose echo term_getansicolors('')
[]
" Details
:verbose echo &term has("gui_running") &t_Co
builtin_gui 1  
```

## Details ##
VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Apr 10 2020 09:45:15)
Included patches: 1-539
Huge version with GTK2 GUI.